armi/reactor/blueprints/componentBlueprint.py

```diff
@@ -42,7 +41,7 @@ def __init__(self, value):
         # note: yamlizable does not call an __init__ method, instead it uses __new__ and setattr
         self.value = value
         if isinstance(value, str):
-            if not components.COMPONENT_LINK_REGEX.search(value):
+            if not component.COMPONENT_LINK_REGEX.search(value):
```

Comment:
Is this regex defined in `armi.reactor.components.component` and then imported into `armi.reactor.components`?

This is not a change I would have expected to see in this PR